clients/src/main/java/org/apache/kafka/common/requests/DescribeConfigsResponse.java

```diff
@@ -17,7 +17,6 @@
 
 package org.apache.kafka.common.requests;
 
-import org.apache.kafka.clients.admin.ConfigEntry;
```

Comment:
this is interesting. we have two almost same `ConfigEntry` ... `org.apache.kafka.clients.admin.ConfigEntry` and `org.apache.kafka.common.requests.DescribeConfigsResponse.ConfigEntry`